### ISSUE TITLE
deprectaed propertie 'openhab.username'  in default config

### ISSUE THREAD

**Issue by CWempe** [openhab/openhab-vscode#278]
I noticed I get a warning in my debug output.

> Usage of deprecated config => openhab.username <= detected.
> Usage of deprecated config => openhab.username <= detected.
> Usage of deprecated config => openhab.username <= detected.

But I could not find this property in my workspace config or my global config.

I did however find it in the default config.
![image](https://user-images.githubusercontent.com/1270359/131907956-11a5770c-4164-43be-a7c3-13d058082b35.png)

![image](https://user-images.githubusercontent.com/1270359/131908002-27a2326b-d1dd-4a35-bda7-faf17117d52e.png)

Is it possible this default entry is causing this warning?

I found it here in the code: https://github.com/openhab/openhab-vscode/blob/6d1d679e6bc3c051aae810b1025c2f445d4f3854/package.json#L143

This is no big deal of cause.
But I think it is strange that a "depricated warning" is caused even the user did not set this property. 😄

**Comment by Confectrician** [comment]
Nope. Those warning should not get triggered with standard values.
I have excluded them in the checks.
(But they are getting removed at some tim ein the future. We just need them to display a deperecation warnin gin the settings editor.)

My guess is that vscode is caching information somewhere/somehow.
Did you try to completely uninstall the extension?
There is a guide/link in the community with some explanations on how to do this.
